docs/getting-started-with-micro/index.md

```diff
@@ -0,0 +1,286 @@
+
+// highlight-next-line
+newTracker('snowplow', 'localhost:9090', {
+  appId: 'my-app-id',
+  plugins: [],
+});
+
+trackPageView();
+enableActivityTracking({
+  heartbeatDelay: 10,
+  minimumVisitLength: 10,
+});
+```
+
+### Checking the results
+
+:::note
+
+The output below is only available in Micro 1.4.0 and above.
+
+:::
+
+Once you have the tracking code and the events are flowing in, you should see something like this in the Micro logs:
+
+```
+[INFO] EventLog - GOOD id:4bfd5b32-d02a-4f83-a731-4339898437e1 app_id:test type:page_view (iglu:com.snowplowanalytics.snowplow/page_view/jsonschema/1-0-0)
+[INFO] EventLog - GOOD id:e7a5c64d-d0f7-48d9-9a50-be044db7d2f6 app_id:test type:page_ping (iglu:com.snowplowanalytics.snowplow/page_ping/jsonschema/1-0-0)
+[INFO] EventLog - GOOD id:1608ca85-f5f9-4948-898a-728aa8f1131b app_id:test type:page_ping (iglu:com.snowplowanalytics.snowplow/page_ping/jsonschema/1-0-0)
+```
+
+This means your tracking is set up correctly and your events are valid (`GOOD`).
+
+If you wish to inspect the events further, you can use [the API](/docs/pipeline-components-and-applications/snowplow-micro/index.md). For example, try:
+
+```bash
+curl localhost:9090/micro/good
+```
+
+### Exporting events to TSV
+
+:::note
+
+This only applies to Micro 1.4.0 and above.
+
+:::
+
+Snowplow pipelines output data in the [_enriched TSV format_](/docs/understanding-your-pipeline/canonical-event/understanding-the-enriched-tsv-format/index.md). Typically, this is picked up by one of our [loaders](/docs/destinations/warehouses-and-lakes/index.md) or by tools such as [Snowbridge](/docs/destinations/forwarding-events/snowbridge/index.md).
+
+With Micro, you can see what your data would look like in this format — useful if you want to test any logic that is parsing this data. 
+
+<CodeBlock language="bash">{
+`docker run -p 9090:9090 snowplow/snowplow-micro:${versions.snowplowMicro} --output-tsv`
+}</CodeBlock>
+
+<details>
+<summary>Output vs logs</summary>
+
+The TSV data will be printed to the [standard output](https://en.wikipedia.org/wiki/Standard_streams#Standard_output_(stdout)). As you saw above, Micro also prints logs, which go into the [standard error stream](https://en.wikipedia.org/wiki/Standard_streams#Standard_error_(stderr)).
+
+Depending on how you are running Micro, you might find the logs distracting. If so, you can turn off event logs with an extra option:
+
+<CodeBlock language="bash">{
+`docker run -p 9090:9090 snowplow/snowplow-micro:${versions.snowplowMicro} --output-tsv \\
+  -Dorg.slf4j.simpleLogger.log.EventLog=off`
+}</CodeBlock>
+
+Or just discard the standard error stream entirely using the syntax appropriate for your shell:
+
+<CodeBlock language="bash">{
+`# for bash
+docker run -p 9090:9090 snowplow/snowplow-micro:${versions.snowplowMicro} --output-tsv \\
+  2>/dev/null`
+}</CodeBlock>
+
+</details>
+
+## Advanced usage
+
+### Adding custom schemas
```

Comment:
I think there's something missing here linking out to custom/self-describing events in addition to just the concept of schemas in the first place - maybe like above just an example tracking block from one of the trackers?